accounts_test.go

```diff
@@ -0,0 +1,72 @@
+package gerrit_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+)
+
+// TestAddSSHKey tests the addition of an SSH key to an account.
+func TestAddSSHKey(t *testing.T) {
+	setup()
+	defer teardown()
+
+	testMux.HandleFunc("/accounts/self/sshkeys", func(w http.ResponseWriter, r *http.Request) {
+		// Ensure the request method is POST
+		if r.Method != http.MethodPost {
+			t.Errorf("Expected POST request, got %s", r.Method)
+		}
+
+		// Ensure Content-Type is text/plain
+		if r.Header.Get("Content-Type") != "text/plain" {
+			t.Errorf("Expected Content-Type 'text/plain', got %s", r.Header.Get("Content-Type"))
+		}
+
+		// Read body and validate SSH key
+		expectedSSHKey := "ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA0T...YImydZAw== john.doe@example.com"
+		body, _ := io.ReadAll(r.Body)
+		receivedSSHKey := strings.TrimSpace(string(body))
+		receivedSSHKey = strings.Trim(receivedSSHKey, `"`)
```

Comment:
This doesn't look right. We want the receivedSSHKey to be the exact `ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA0T...YImydZAw== john.doe@example.com` instead of `"ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA0T...YImydZAw== john.doe@example.com"`.